tests/keras/backend/backend_test.py

```diff
@@ -67,7 +71,8 @@ def parse_shape_or_val(shape_or_val):
         return shape_or_val, np.random.random(shape_or_val).astype(np.float32) - 0.5
 
 
-def assert_list_pairwise(z_list, shape=True, allclose=True, itself=False, atol=1e-05):
+def assert_list_pairwise(z_list,
+                         shape=True, allclose=True, itself=False, atol=1e-05):
```

Comment:
Style: if you're going to break a signature or a call like this, prefer stacking the arguments for better readability, like

```python
def assert_list_pairwise(z_list,
                         shape=True,
                         allclose=True):
```

This is applicable in several places in this PR

Reply:
@fchollet Thanks for the feedback.

That's actually the style I do prefer. I was just following the example of `check_two_tensor_operation` (and others). For consistency I changed these, too.